mirrord/layer/src/tcp_steal/http.rs

```diff
@@ -84,6 +85,7 @@ pub(super) trait HttpV: Sized {
 
     /// Sends the [`HttpRequest`] through `Self::sender`, converting the response into a
     /// [`HttpResponse`].
+    #[tracing::instrument(level = "trace", skip(self))]
```

Comment:
skip `HttpRequest` or it's without data?

Reply:
Yeah, the `body` is logged as just `bytes (length): 100` for example.